app/Http/Controllers/PostsController.php

```diff
@@ -9,7 +9,6 @@
 use Rogue\Repositories\SignupRepository;
 use Rogue\Http\Transformers\PostTransformer;
 use Rogue\Http\Controllers\Traits\FiltersRequests;
-use Rogue\Http\Controllers\Legacy\Two\ApiController;
```

Comment:
Does this still need to include this at its new location or no since they're in the same directory now?

Reply:
I don't think it does since they're in the same directory now!